android/java/apk_for_test.flags

```diff
@@ -507,8 +507,6 @@
 }
 
 -keep class org.chromium.chrome.browser.omnibox.suggestions.DropdownItemViewInfoListBuilder {
-    *** mDropdownHeight;
```

Comment:
I suppose changes in `brave/build/android/bytecode/java/org/brave/bytecode/BraveDropdownItemViewInfoListBuilderClassAdapter.java` are not necessary as well anymore?

Reply:
Yes, thanks for pointing out. removing now